src/main/java/org/alliancegenome/curation_api/model/entities/Molecule.java

```diff
@@ -47,12 +49,14 @@
 	
 	@KeywordField(aggregable = Aggregable.YES, sortable = Sortable.YES, searchable = Searchable.YES)
 	@JsonView({View.FieldsOnly.class})
+	@Column(columnDefinition="TEXT")
 	private String smiles;
 	
-	@ManyToMany
+	@ElementCollection
+	@JsonView(View.FieldsAndLists.class)
 	@JoinTable(indexes = @Index( columnList = "molecule_curie"))
-	@JsonView({View.FieldsAndLists.class})
-	private List<Synonym> synonyms;
+	@Column(columnDefinition="TEXT")
```

Comment:
These `@Column(columnDefinition="TEXT")` are only needed if the field is going to be longer than 255 charactors, otherwise it's better to leave it out.

Reply:
Yeah, I had removed that but there are indeed synonyms > 255 chars